src/sagas.js

```diff
@@ -30,5 +44,6 @@ export function* rootSaga() {
     notesInit(),
     noteMindMapInit(),
     notesContentEditorInit(),
+    fork(attachmentsInit),
```

Comment:
Why are we doing all inits except this one without `fork()`? :)
What's the point of `fork`ing the init, if it's only doing 
```
 yield all([
    takeEvery(Actions.addUploadingFiles, addUploadingFilesSaga),
    takeEvery(Actions.retryFileUpload, retryFileUploadSaga),
  ]);
```
?
It's only initializing the listeners, it's not doing any job?
If we fork the initialization, does it mean that the user can click the upload button before the saga initialized, so the button won't have any effect? I.e. if `addUploadingFiles` action would be fired before saga has started listening to this event?